database/src/migrations/20200810113233_init.ts

```diff
@@ -0,0 +1,37 @@
+import Knex from 'knex';
+
+const DB_SCHEMA = process.env.DB_SCHEMA || 'invasivesbc';
+
+/**
+ * Create the `DB_SCHEMA` schema.
+ *
+ * @export
+ * @param {Knex} knex
+ * @return {*}  {Promise<void>}
+ */
+export async function up(knex: Knex): Promise<void> {
+  await knex.raw(`
+    set schema '${DB_SCHEMA}';
+    set search_path = ${DB_SCHEMA},public;
+
+    CREATE SCHEMA IF NOT EXISTS invasivesbc;
+
+    ALTER SCHEMA invasivesbc OWNER TO invasivebc;
+  `);
+}
+
+/**
+ * Drop the `DB_SCHEMA` schema.
+ *
+ * @export
+ * @param {Knex} knex
+ * @return {*}  {Promise<void>}
+ */
+export async function down(knex: Knex): Promise<void> {
+  await knex.raw(`
+    set schema '${DB_SCHEMA}';
+    set search_path = ${DB_SCHEMA},public;
+
+    DROP SCHEMA IF EXISTS ${DB_SCHEMA}.invasivesbc;
```

Comment:
This is breaking the downgrade.  Just need to remove `.invasivesbc`
```suggestion
    DROP SCHEMA IF EXISTS ${DB_SCHEMA};
```

Reply:
Nice catch!

Also, never seen this 'commit suggestion' option before, thats interesting